packages/viewer/src/components/viewer-markup/viewer-markup-components.tsx

```diff
@@ -0,0 +1,236 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { FunctionalComponent, h } from '@stencil/core';
+import { Point, Angle, Rectangle } from '@vertexvis/geometry';
+import { DeviceSize } from '../../lib/device';
+import { getBoundingBox2dAnchorPosition } from './utils';
+
+export interface SvgShadowProps {
+  id: string;
+}
+
+export const SvgShadow: FunctionalComponent<SvgShadowProps> = ({ id }) => {
+  return (
+    <filter id={id} filterUnits="userSpaceOnUse">
+      <feGaussianBlur in="SourceAlpha" stdDeviation="2" />
+      <feOffset dx="0" dy="1" result="offsetblur" />
+      <feFlood flood-color="#000000" flood-opacity="0.25" />
+      <feComposite in2="offsetblur" operator="in" />
+      <feMerge>
+        <feMergeNode />
+        <feMergeNode in="SourceGraphic" />
+      </feMerge>
+    </filter>
+  );
+};
+
+export interface BoundingBoxAnchorProps {
+  id?: string;
+  center: Point.Point;
+  angle?: number;
+  width?: number;
+  height?: number;
+  onGrab?: (event: PointerEvent) => void;
+}
+
+export const BoundingBoxAnchor: FunctionalComponent<BoundingBoxAnchorProps> = ({
+  id,
+  angle,
+  center,
+  width,
+  height,
+  onGrab,
+}) => {
+  return (
+    <rect
+      id={id}
+      class="bounds-rect"
+      height={height ?? 8}
+      width={width ?? 8}
+      x={center.x - (width != null ? width / 2 : 4)}
+      y={center.y - (height != null ? height / 2 : 4)}
+      transform={angle ? `rotate(${angle},${center.x},${center.y})` : undefined}
+      onPointerDown={onGrab}
+    />
+  );
+};
+
+export interface BoundingBox1dProps {
+  start: Point.Point;
+  end: Point.Point;
+  deviceSize?: DeviceSize;
```

Comment:
This would be better customized from the applicationn through CSS variables.

Reply:
For sure, this was related to challenges with styling the SVG anchors. This should go away with swapping to HTML anchors